src/plugins/event-plugins/JsErrorPlugin.ts

```diff
@@ -90,4 +98,17 @@ export class JsErrorPlugin implements Plugin {
             this.promiseRejectEventHandler
         );
     }
+
+    private isErrorInIgnoreList(message: string): boolean {
+        if (this.ignoredErrors.has(message)) {
```

Comment:
Yeah, as mentioned in the previous comment above, the idea was to avoid going through a large list of regex that the customer passes in, but I do think this may not be needed since I've found some posts that it takes up to couple thousand inputs to reach 1 second of process time. Also great point, I personally have not thought of that case (and that is the biggest point proving we should take it out)

Reply:
Sounds good 👍